pkgs/applications/science/biology/hh-suite/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, xxd
+, enableMpi ? false
+, mpi
+, openmp
+}:
+stdenv.mkDerivation rec {
+  pname = "hh-suite";
+  version = "3.3.0";
+
+  src = fetchFromGitHub {
+    owner = "soedinglab";
+    repo = "hh-suite";
+    rev = "v${version}";
+    hash = "sha256-kjNqJddioCZoh/cZL3YNplweIGopWIGzCYQOnKDqZmw=";
+  };
+
+  nativeBuildInputs = [ cmake xxd ];
+  cmakeFlags = [ "-DHAVE_SSE2=1" ]
```

```suggestion
  cmakeFlags = lib.optional stdenv.hostPlatform.isx86 "-DHAVE_SSE2=1"
```
aarch64 does not support that I just remembered

---

Thanks!
Also, I added the flag for ARM.